server/views/pages/prisons/viewSingleSessionTemplate.njk

```diff
@@ -0,0 +1,200 @@
+{% extends "../../partials/layout.njk" %}
+{%- from "govuk/components/tag/macro.njk" import govukTag -%}
+{%- from "govuk/components/button/macro.njk" import govukButton -%}
+{%- from "moj/components/banner/macro.njk" import mojBanner -%}
+{%- from "moj/components/primary-navigation/macro.njk" import mojPrimaryNavigation -%}
+{%- from "govuk/components/summary-list/macro.njk" import govukSummaryList -%}
+{%- from "moj/components/page-header-actions/macro.njk" import mojPageHeaderActions -%}
+{%- from "govuk/components/back-link/macro.njk" import govukBackLink -%}
+
+{% set pageHeaderTitle = sessionTemplate.name %}
+{% set pageTitle = applicationName + " - " + pageHeaderTitle %}
+
+{% set primaryNavItems = [
+  {
+    text: "Home",
+    href: "/"
+  },
+  {
+    text: "Prisons",
+    href: "/prisons",
+    active: true
+  }
+  ] %}
+
+{% set backLinkHref = "/prisons/" + sessionTemplate.prisonId + "/session-templates" %}
+
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ prisonName }}</span>
+        {{ pageHeaderTitle }}
+      </h1>
+
+      {% include "partials/errorSummary.njk" %}     
+
+      {% if message | length %}
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            {{ mojBanner({
+              type: 'success',
+              text: prisonName + " has been " + message + '.',
+              iconFallbackText: 'Success'
+            }) }}
+          </div>
+        </div>
+      {% endif %}
+
+      {%- set locationGroups -%}
+        {%- if sessionTemplate.permittedLocationGroups.length !== 0 -%}
```

Comment:
This works fine, but you can just use:
`{%- if sessionTemplate.permittedLocationGroups.length -%}`
or
`{%- if sessionTemplate.permittedLocationGroups | length -%}`.

These work because Nunjucts considers a length greater than zero to be 'truthy'.